docs/index.rst

```diff
@@ -23,6 +24,7 @@ Blocks is developed in parallel with Fuel_, a dataset processing framework.
 .. _mailing list: https://groups.google.com/forum/#!forum/blocks-users
 .. _making a GitHub issue: https://github.com/mila-udem/blocks/issues/new
 .. _Fuel: https://github.com/mila-udem/fuel
+.. _examples: https://github.com/mila-udem/blocks-examples
```

Comment:
You don't need to define this reference again, Shynx doesn't like it. https://github.com/mila-udem/fuel/pull/232#discussion_r39973585